ui.apps/src/main/content/jcr_root/apps/hclecomm/config/com.adobe.acs.commons.errorpagehandler.impl.ErrorPageHandlerImpl.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OsgiConfig"
+    enabled="{Boolean}true"
+    error-page.fallback-name="500"
```

Comment:
@rakeshkumar-verma : where have you configured 404 error ?

Reply:
@rishabh-behl  Based on error code corresponding error page will be picked from .../errorPages  page.